binaries/agent-image

```diff
@@ -0,0 +1 @@
+datadog/agent:7
```

Comment:
Hmm, this was removed from `.gitignore` in https://github.com/DataDog/system-tests/commit/ea77db96e290a4dc3b078481b35dfab5979594fe

Any reason? @TonyCTHsu

Reply:
was a typo I guess. Putting it back